src/components/Switcher.vue

```diff
@@ -45,7 +45,7 @@ export default {
       });
       this.options[index].checked = true;
       let title = this.options[index].title;
-      if (title === "title" || title === "genre") {
+      if (title === "title" || title === "genres") {
```

Comment:
Such application-specific values as "title" and "genres" that are used in case-sensitive logic should be put in the app configuration directory for ease of control.
E.g there could be a config directory at the root level that contains such things.